vlib/v/tests/bench/channel_bench_go.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
I think that the channel tests should be under vlib/sync/ for now, because the channels implementation is also there.